tests/conftest.py

```diff
@@ -2242,3 +2250,38 @@ def mock_config(rnafusion_dir: Path, rnafusion_case_id: str) -> None:
     Path(rnafusion_dir, rnafusion_case_id, f"{rnafusion_case_id}_samplesheet.csv").touch(
         exist_ok=True
     )
+
+
+@pytest.fixture
+def store_with_sequencing_metrics(store: Store):
+    """Return a store with multiple samples with sample lane sequencing metrics."""
+
```

Comment:
Should not the populating of the store be done in a function in the StoreHelper class to keep it DRY, consistent and reusable when building more complex stores?

Reply:
I think that can be done if the need arises, I think the existing store is fine for most test cases I can think of at the moment! Let's add it when it is actually needed